index.html

```diff
@@ -1,28 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-    <meta charset="utf-8">
-		<title>Document Title</title>
-		
-		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
-		<link rel="stylesheet" href="css/style.css">
-	</head>
-	<body>
+
+<head>
+	<meta charset="utf-8">
+	<title>Spirit Dunkin - Hippie Florist</title>
+
+	<!-- external CSS link -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+	<h1>Spirit Dunkin</h1>
+	<p>Flower Shop Owner, Williamsburg</p>
+		<img src="images/orchids.jpg" />
```

Comment:
The indentation here and a lot of other places in your HTML is very erratic. 

Please keep your indentation tidy as it makes it very difficult for others to read and understand your code!

It's best to do this as you go, but worst-case, you can always choose `Auto Indent` in atom to indent everything correctly.